src/metabase/api/card.clj

```diff
@@ -738,8 +739,9 @@ saved later when it is ready."
         (when-let [cards-without-position (seq (for [card cards
                                                      :when (not (:collection_position card))]
                                                  (u/the-id card)))]
-          (db/update-where! Card {:id [:in (set cards-without-position)]}
-            :collection_id new-collection-id-or-nil))))))
+          (t2/update! (t2/table-name Card)
```

Comment:
What's the explanation for why this uses table-name? Just performance?

Reply:
I'm not 100% sure but without it moving cards will get a duplicated card_id error
`ERROR: duplicate key value violates unique constraint \"report_card_pkey\"\n  Detail: Key (id)=(590) already exists`

try running this test `test-that-we-can-bulk-remove-some-cards-from-a-collection`

probably have something to do with the fact that this is being called under a transaction.